src/worker.h

```diff
@@ -62,7 +59,16 @@ namespace NLV {
       T *child = Nan::ObjectWrap::Unwrap<T>(childObject);
       NLVObjectBasePtr *childPtr = new NLVObjectBasePtr(child);
       child->SetParentReference(childPtr);
-      //parent_->children_.push_back(childPtr);
+      auto parent = ParentClass::Unwrap(parentObject);
+      if(parent) {
+        parent->children_.push_back(childPtr);
+      }
+      Nan::TryCatch try_catch;
+      if(try_catch.HasCaught()) {
```

Comment:
As I understand, just creating a `Nan::TryCatch` should catch exceptions. https://github.com/nodejs/nan/blob/master/doc/errors.md#api_nan_try_catch - and exceptions can be caught inside `ParentClass::Unwrap` for example when a value is not of `ParentClass` type. See https://github.com/hooklift/node-libvirt/pull/74/files/38bff64c7c6f81bbafa0461bb3db82e4f1a6ab3e#diff-c679822ceba701f71305a5025962b67aR96